package.json

```diff
@@ -32,6 +32,7 @@
     "jasmine-spec-reporter": "^4.2.1",
     "protractor": "^5.4.2",
     "save-dev": "^2.0.0",
+    "selenium-webdriver": "^4.0.0-alpha.5",
```

Comment:
why do you need this?

Reply:
It was added when I updated the webdriver. It has been removed.